### DISCUSSION TITLE
Please help me change the time zone Помогите пожалуйста сменить часовой пояс

### DISCUSSION THREAD

**Discussion by Horman-Oladushek** [IATkachenko/HA-YandexWeather#121] [Q&A] [answered]
Eng: I want to set up data about when the weather was updated to my time zone, but I didn't figure out how to do it. Can you advise/direct how to change from +0 GTM to +5 GTM. Greetings from Oren

Rus: Я хочу настроить данные о том, когда было обновлена погода на свой часовой пояс, но не понял как это сделать. Можете посоветовать/направить как нужно сделать, чтобы изменить с +0 GTM было +5 GTM. Привет из Орена

**Comment by Horman-Oladushek** [comment]
Забыл уточнить, что информация о том, когда информация обновлена отправляется мне в телеграмм при помощи бота

**Comment by IATkachenko** [comment]
@Horman-Oladushek, привет!

Вот [тут](https://github.com/IATkachenko/HA-YandexWeather/blob/1c1d2d662e19b3ed2412aedda1357efa8c9c6a71/custom_components/yandex_weather/updater.py#L246) извлекается таймзона HA и к ней приводится время из текущей погоды и прогноза. Соответственно, нужно в HA выставить правильную таймзону и все должно заработать. Если нет -- открой, пожалуйста issue, будем разбираться.

ps. Do you prefer English or Russian communication?

**Reply by Horman-Oladushek** [reply]
Привет, русский) как попробую, кину ответ

**Reply by Horman-Oladushek** [reply]
можно пожалуйста для меня тупого, как установить определенную таймзону в HA?

**Reply by IATkachenko** [reply, marked as answer]
> можно пожалуйста для меня тупого, как установить определенную таймзону в HA?

Настройки -> Система -> Общие -> Часовой пояс
![изображение](https://github.com/user-attachments/assets/408dde7e-02c8-4bbb-8f43-f3c1477aac12)

**Reply by Horman-Oladushek** [reply]
все, разобрался, спасибо!
